helfi_features/helfi_react_search/config/install/field.storage.paragraph.field_event_count.yml

```diff
@@ -0,0 +1,48 @@
+uuid: 50155258-a9bc-4108-9264-a3d0b3c5ad2d
+langcode: en
+status: true
+dependencies:
+  module:
+    - options
+    - paragraphs
+id: paragraph.field_event_count
+field_name: field_event_count
+entity_type: paragraph
+type: list_integer
+settings:
+  allowed_values:
```

Comment:
`allowed_values` can be empty array when `allowed_values_function` is set. 
Like so:`   allowed_values: {  }`